metricbeat/module/kibana/cluster_actions/cluster_actions.go

```diff
@@ -79,13 +69,32 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 // It returns the event which is then forward to the output. In case of an error, a
 // descriptive error must be returned.
 func (m *MetricSet) Fetch(r mb.ReporterV2) (err error) {
+	if err = m.validate(); err != nil {
+		return err
+	}
+
 	if err = m.fetchMetrics(r); err != nil {
 		return fmt.Errorf("error trying to get cluster action data from Kibana: %w", err)
 	}
 
 	return nil
 }
 
+func (m *MetricSet) validate() error {
+	kibanaVersion, err := kibana.GetVersion(m.actionsHTTP, kibana.ClusterActionsPath)
+	if err != nil {
+		return err
+	}
+
+	isMetricsAPIAvailable := kibana.IsActionsAPIAvailable(kibanaVersion)
+	if !isMetricsAPIAvailable {
+		const errorMsg = "the %v node actions is only supported with Kibana >= %v. You are currently running Kibana %v"
+		return fmt.Errorf(errorMsg, m.FullyQualifiedName(), kibana.ActionsAPIAvailableVersion, kibanaVersion)
```

Comment:
Since this will be run on every fetch and it is not a fatal error, I think we should log (at debug level) instead of returning an error which will potentially be log spam for unsupported kibanas

Reply:
Log noise has been a problem in other modules, for example in beats https://github.com/elastic/beats/pull/34460, and a pattern exists to reduce occurrences of these log unhelpful messages (see https://github.com/elastic/beats/blob/main/metricbeat/module/elasticsearch/enrich/enrich.go#L77). It would make sense to adopt this pattern here, wdyt ?

Reply:
Sound good! I'll do that.